src/argilla/server/server.py

```diff
@@ -212,9 +212,10 @@ async def check_telemetry():
             print(message, flush=True)
 
 
-def configure_database(app: FastAPI):
-    get_db_wrapper = contextlib.asynccontextmanager(get_async_db)
+_get_db_wrapper = contextlib.asynccontextmanager(get_async_db)
```

Comment:
This has to be here, otherwise it cannot be mocked. I tried to mock `get_async_db`, but I couldn't mock it directly using `argilla.server.server.get_async_db`. I was able to mock it using `argilla.server.database.get_async_db` but that caused the rest of the unit tests to fail :)

Reply:
Make sense to move this function to `argilla.server.database` and use it here?

Reply:
right now is only being used here and I think it won't be needed in any other part of the code. I would keep it here to avoid confusions with `get_async_db` function.